test/collapsers/javascript.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+var assert = require('power-assert');
+var Bluebird = require('bluebird');
+var describe = require('mocha').describe;
+var it = require('mocha').it;
+var collapser = require('../../lib/collapsers/javascript');
+
+describe('JavaScript collapser', function () {
+  it('should minify JavaScript', function () {
```

Comment:
There's an expectation that UglifyJS works correctly, but the exact output can change from version to version.